### PR TITLE
Updating PR Check Action

### PR DESCRIPTION
**Pull request by justetz**

### CODE REVIEW

File: .github/workflows/master-pr-check-version.yml [hunk 1]
```diff
@@ -13,11 +13,19 @@ jobs:
...
     - name: Check if version changed
       run: |
-        version_changed=`git diff HEAD..origin/master package.json | grep version | wc -l | tr -d '[:space:]'`
+        comparison_branch="HEAD..origin/master"
+        version_changed=`git diff $comparison_branch package.json | grep version | wc -l | tr -d '[:space:]'`
         if [ $version_changed -gt 1 ]; then
-          echo "Version has been modified; good to go!"
-          exit 0
+            echo "Versions are different; verifying that the version increased"
+            version_increased=`git diff $comparison_branch package.json | grep version | awk -F\" '{split($4, old, "."); getline; split($4, new, "."); if(new[1] > old[1] || (new[1] == old[1] && new[2] > old[2]) || (new[1] == old[1] && new[2] == old[2] && new[3] > old[3])) { print 1 } else { print 0 } }'`
```
Comment (ddbruce) [inline .github/workflows/master-pr-check-version.yml]: Any way we can split this line for readability?

Comment (justetz) [reply .github/workflows/master-pr-check-version.yml]: I believe so but I don't know specifically how

Comment (justetz) [reply .github/workflows/master-pr-check-version.yml]: Done

**Comment by ddbruce** [comment]
Looks awesome @justetz! Thanks!

**Pull request merged**